### PR TITLE
Fix part of #1824 : Full UI: Tablet: StoryActivity Landscape + Portrait [WIP] [DO NOT MERGE]

### PR DESCRIPTION
**Pull request by prayutsu**
## Explanation

Fix part of #1824 
Added layouts for landscape version of tablet layouts and matched UI with the mock

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by prayutsu** [comment]
@rt4914 PTAL

**Comment by rt4914** [comment]
@prayutsu You have mentioned that you have "matched UI with the mock" , but your screenshot do not match with the mock. So what exactly you are trying to mention here. Thanks.

**Comment by prayutsu** [comment]
> @prayutsu You have mentioned that you have "matched UI with the mock" , but your screenshot do not match with the mock. So what exactly you are trying to mention here. Thanks.

I am trying to mention that by changing padding, the size of the recyclerview item is matching with that in the mock.

**Comment by rt4914** [comment]
> > @prayutsu You have mentioned that you have "matched UI with the mock" , but your screenshot do not match with the mock. So what exactly you are trying to mention here. Thanks.
> 
> I am trying to mention that by changing padding, the size of the recyclerview item is matching with that in the mock.

Okay, that will actually be verified in the final implementation of highfi work. Highfi work should not be done in parts like this, instead everything related to highfi should be in a single PR for a particular screen/component. 

This can be merged now, right?

**Comment by prayutsu** [comment]
> Okay, that will actually be verified in the final implementation of highfi work. Highfi work should not be done in parts like this, instead everything related to highfi should be in a single PR for a particular screen/component.
> 
> This can be merged now, right?

Can you please summarize once, what else needs to be done to close #1824?

**Comment by rt4914** [comment]
> > Okay, that will actually be verified in the final implementation of highfi work. Highfi work should not be done in parts like this, instead everything related to highfi should be in a single PR for a particular screen/component.
> > This can be merged now, right?
> 
> Can you please summarize once, what else needs to be done to close #1824?

@prayutsu Any updates on this?

**Comment by anandwana001** [comment]
@prayutsu You need to update the branch with `develop` before working more on this PR as we had updated the package names.

**Comment by prayutsu** [comment]
> @prayutsu You need to update the branch with `develop` before working more on this PR as we had updated the package names.

Okay, I'll update it

**Comment by prayutsu** [comment]
@rt4914 I tried making a line (currently roughly similar to the one in mock). I am stuck on making it of the desired height. 

(Please refer to the self-reviewed comment)

If I set the ```layout_height``` to ```match_parent``` then the following result is obtained - 
Screenshot_1601142183

**Comment by prayutsu** [comment]
If I set the ```layout_height``` to ```200dp``` then the following result is obtained - 

Screenshot_1601142271

File: app/src/main/res/layout-sw600dp-port/story_chapter_view.xml [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+      <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:clipToPadding="true"
+        android:contentDescription="@{viewModel.name}">
+
+        <org.oppia.android.app.customview.VerticalDashedLine
+          android:layout_width="5dp"
+          android:id="@+id/line"
+          android:layout_height="200dp"
```
Comment (prayutsu) [inline app/src/main/res/layout-sw600dp-port/story_chapter_view.xml:48]: If I am setting the layout_height here to match_parent then it's not working as expected.

**Comment by prayutsu** [comment]
Why ```onMeasure``` is not working properly?

**Comment by prayutsu** [comment]
Also, it does not show any views in the design mode fir ```story_chapter_view``` which makes it really difficult to see the results because every time I make some changes, I have to fire up the emulator again. Can this be fixed?
Screenshot from 2020-09-26 23-11-09

**Comment by anandwana001** [comment]
Try this [reference](https://stackoverflow.com/questions/55135893/how-to-implement-vertical-dotted-line-below-image-in-android)

**Comment by prayutsu** [comment]
@anandwana001 The reference contains really a beautiful example but we want "rounded corners" of the dashes of the line and this reference contains simple rectangular dashes.
https://xd.adobe.com/view/d405de00-a871-4f0f-73a0-f8acef30349b-a234/screen/d70a3ade-826a-4ac6-95e6-907a24d7b896/

**Comment by anandwana001** [comment]
> @anandwana001 The reference contains really a beautiful example but we want "rounded corners" of the dashes of the line and this reference contains simple rectangular dashes.
> https://xd.adobe.com/view/d405de00-a871-4f0f-73a0-f8acef30349b-a234/screen/d70a3ade-826a-4ac6-95e6-907a24d7b896/

That something we can modify and as I mentioned, you can check this for just a reference.

**Comment by rt4914** [comment]
@anandwana001 If this PR is ready from your side, should I review it?

**Comment by anandwana001** [comment]
> @anandwana001 If this PR is ready from your side, should I review it?
> 

I had a discussion with the @prayutsu regarding this. It will be good if you give a look, although this PR is WIP.

**Comment by rt4914** [comment]
@prayutsu Any updates after our discussion?

**Pull request closed without merging**